jmeter-java-dsl-datadog/src/main/java/us/abstracta/jmeter/javadsl/datadog/DatadogBackendListener.java

```diff
@@ -58,7 +58,7 @@ public static DatadogBackendListener datadogListener(String apiKey) {
    * @param site specifies the site to use. When no site is specified, US1 is used by default.
    * @return the listener for further configuration and usage.
    */
-  public DatadogBackendListener site(DatadogSite site) {
```

Comment:
In general we are not adding the "set" prefix to methods for setting properties of test elements to keep them short and at this point follow the general convention. 

Additionally, I think that in this case adding Datadog to the method is redundant, since once you are setting a datadog property named site, I think is quite natural to think is the Datadog Site. I do agree though that datadog site is a little too generic, and doesn't easily reflect the purpose of the variable, maybe something like region would be better, but site is the name that datadog has assigned to it, so as not to confuse too much users we kept with that name. 

Finally, this is a non backwards change: if some users are already using the existing method, then their test plans will stop working unless they change the method. In general we avoid making these changes unless it was a very recent version that we hope none to too few users are using it. If we ever come to a change like this, and we really want to make it, then we put it in a backlog of changes to be done on next major version (eg 2.0).